README.md

```diff
@@ -1,22 +1,6 @@
-# .NET Install Tool for Extension Authors
+# .NET Runtime and SDK Installation Tools
 
-[![Version](https://vsmarketplacebadge.apphb.com/version/ms-dotnettools.vscode-dotnet-runtime.svg)](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-runtime) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-dotnettools.vscode-dotnet-runtime.svg)](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-runtime)
-
-This extension allows acquisition of the .NET Core runtime specifically for Visual Studio Code extension authors. This tool is intended to be leveraged in extensions that are written in .NET and require .NET to boot pieces of the extension (e.g. a language server). The extension is not intended to be used directly by users to install .NET for development.
-
-## Goals: Acquiring .NET Core for extensions
-
-Extension authors do not know if the .NET core runtime is installed on the target machine. Existing solutions have a number of challenges:
-
-1. **Duplication of .NET Core runtimes and slow updates**: Currently, each extension is acquiring its own copy of .NET core at ~30mb each.
-2. **Clean up**: When extensions install .NET Core in a non-VSCode folder location it is likely to be left behind.
-3. **Servicing and floating versions**: It is difficult to ensure that extensions will use the latest releases, particuarly without re-shipping.
-4. **Corrupted installations**: Corrupted installations can arise when VS Code is shut down mid-download or unzip.
-5. **Network security policies**: Alternative installation methods may result in errors due to blocking from network security policies.
-6. **Locked down environments**: Some developers are unable to freely install software, requiring the ability to install extensions manually via a VSIX.
-7. **Missing dependencies**: Users may run into situations where .NET Core cannot run as-is, requiring the installation of missing pieces.
-
-This extension attempts to solve the above issues.
+This repo contains two VS Code extensions, [vscode-dotnet-runtime](vscode-dotnet-runtime-extension/README.md) and [vscode-dotnet-sdk](vscode-dotnet-sdk-extension/README.md). These extensions can be used to install the .NET runtime and SDK respectively and are meant to be leveraged by other extensions which depend on these assets.
```

Comment:
```suggestion
This repo contains two VS Code extensions, [vscode-dotnet-runtime](vscode-dotnet-runtime-extension/README.md) and [vscode-dotnet-sdk](vscode-dotnet-sdk-extension/README.md). The [vscode-dotnet-runtime](vscode-dotnet-runtime-extension/README.md) can be used to install the .NET runtime and is meant to be leveraged by other extensions which depend on the runtime. The.[vscode-dotnet-sdk](vscode-dotnet-sdk-extension/README.md) is a special install for internal features and not designed to be used by other extensions because it will conflict with existing SDK installations on the users machine. 
```